package.json

```diff
@@ -80,7 +80,19 @@
     "webpack-dev-middleware": "^1.11.0",
     "webpack-hot-middleware": "^2.20.0"
   },
-  "keywords": [],
+  "keywords": [
+    "react"
```

Comment:
@mateusz-brainhub 

here you missed a comma

Reply:
done

Reply:
Piotr could you have a look if now is OK?

Reply:
@mateusz-brainhub 

I asked a question why you had changed `dist` into `lib`